<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>921</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>881</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>LED控制</string>
   </property>
   <widget class="QPushButton" name="btn_led1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_led2">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_led1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_led2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_led3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_led4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcd_vol">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>AD模拟量绘图</string>
   </property>
   <widget class="QPushButton" name="btn_adSwitch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="IptextBrowser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="PorttextBrowser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>远程连接控制</string>
   </property>
   <widget class="QPushButton" name="TcpConnectBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>120</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>程序退出</string>
   </property>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>lcd_vol</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>IptextBrowser</zorder>
  <zorder>PorttextBrowser</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
